static/app/utils/profiling/flamegraph.ts

```diff
@@ -37,8 +38,8 @@ const leftHeavyTreeSort = makeTreeSort(sortByTotalWeight);
 
 export class Flamegraph {
   profile: Profile;
+  metadata: ProfileGroup['metadata'] = {};
   frames: ReadonlyArray<FlamegraphFrame> = [];
-  profileIndex: number;
```

Comment:
I removed this from the flamegraph model as we don't need it. The source of truth for this should be the provider, which initializes it's state the response.